package.json

```diff
@@ -1,12 +0,0 @@
-{
-  "name": "simple-enum",
-  "version": "1.0.0",
-  "description": "A very, very tiny JavaScript enum implementation",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/tolgaek/node-enum"
-  },
-  "author": "Tolga Ekmen <tolga.ekmen@gmail.com>",
-  "license": "MIT"
-}
```

Comment:
Can you help me understand why this metadata was removed? It seems useful to maintain attribution and description